S4S scenario (#8503)
cloudflare/cloudflare-docs: content/ssl/edge-certificates/caa-records.md

- You uploaded your own custom origin server certificate (not provisioned by Cloudflare).
- That certificate was issued by a CA (not self-signed).
- Your domain is on a full setup (not a [CNAME setup](/dns/zone-setups/partial-setup)).

> - When adding new Custom Hostname and they have an existing CAA records. In this case, you should remove the existing CAA records or add the missing CAA record.

Reviewer: ```suggestion
- When adding new [Custom Hostname](/cloudflare-for-platforms/cloudflare-for-saas/start/getting-started/) and your customer has existing CAA records. In this case, ask your customer to remove the existing CAA records or add the missing CAA record.
```
